hack/local-up-cluster.sh

```diff
@@ -750,7 +750,7 @@ function start_kubelet {
 
     # shellcheck disable=SC2206
```

Comment:
we should not have inline shellcheck disable comments :/
this lint also catches legitimate bugs

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2206

Reply:
ack, i'll get rid of it when i get a chance

Reply:
will come back to this one later.